HunterPie.Native/HunterPie.Native.vcxproj

```diff
@@ -71,18 +71,18 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
-    <LinkIncremental>true</LinkIncremental>
```

Comment:
`/INCREMENTAL` is already among the default msvc project configuration. You can double confirm from property page -> Linker. If you create a new cpp project, there is no such property explicitly specified, either.

btw this project is effectively not enabling `/INCREMENTAL` because we are using a RELEASE distro of libMinHook.

```
1>RiseUtils.cpp
1>libMinHook.x64.lib(hook.obj) : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linker performance
1>LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/LTCG' specification
```